sdks/go/pkg/beam/core/typex/special.go

```diff
@@ -64,6 +65,23 @@ type Window interface {
 	Equals(o Window) bool
 }
 
+type Timing string
+
+const (
+	EARLY   Timing = "EARLY"
+	ON_TIME Timing = "ON_TIME"
+	LATE    Timing = "LATE"
+	UNKNOWN Timing = "UNKNOWN"
```

Comment:
Despite these being constants, it's not idiomatic go to use All caps for constants. 

I also recommend prefixing what these are related to eg. PaneEarly, PaneOnTime, PaneLate, PaneUnknown 

Mostly so they're adjacent in Go Doc. vs suffixing, which is a bit more readable.

We can only get away without prefixing because we don't have a "pane" package which would delare what they're for earlier. In this case we shouldn't have a pane package, it would be too small for practical uses.

This is because the typex package has a pretty grab bag set of values, so the enumerations need to be made clearer for their use elsewhere.

Reply:
Got it.